#! /bin/sh /usr/share/dpatch/dpatch-run
## 00_gcc43fix.dpatch by Kevin Coyner <kcoyner@debian.org>
##
## DP: in csstidy/misc.cpp, add uncaught missing #include
## DP: closes bug #455123

@DPATCH@
diff -urNad csstidy-1.4~/csstidy/misc.cpp csstidy-1.4/csstidy/misc.cpp
--- csstidy-1.4~/csstidy/misc.cpp	2007-07-10 10:20:54.000000000 -0400
+++ csstidy-1.4/csstidy/misc.cpp	2008-02-24 20:49:26.000000000 -0500
@@ -16,6 +16,7 @@
  * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
  */
  
+#include <cstring>
 #include "csspp_globals.hpp"
 
 bool escaped(const string &istring, const int pos) 
